tests/Service/ApiKeyGeneratorTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Tests\Service;
+
+use App\Service\ApiKeyGenerator;
+use PHPUnit\Framework\TestCase;
+use function strlen;
+
+class ApiKeyGeneratorTest extends TestCase
+{
+    public function test_should_count_30()
+    {
+        $generator = new ApiKeyGenerator();
```

Comment:
Możesz sobie stworzyć taki obiekt w setUp (taki konstruktor dla testów) -> wtedy będziesz miał czyste testy

Reply:
Almost. Teraz mozesz usunąć tą zmienną i użyć $this->generator w teście ;p